# Array of 3D Grasp Detections
Header header
GraspDetection3D[] detections